boards/px4/fmu-v5/default.cmake

```diff
@@ -63,9 +63,11 @@ px4_add_board(
 		uavcan
 	MODULES
 		airspeed_selector
+		angular_velocity_controller
```

Comment:
I'll remove this stuff here - didn't realize there was already a `ctrlalloc` target